Description: Don't enable backlight helper if GUdev is not available
Author: Michael Biebl <biebl@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=676407
Index: gnome-settings-daemon-3.4.2/configure.ac
===================================================================
--- gnome-settings-daemon-3.4.2.orig/configure.ac	2012-05-20 03:37:53.546282530 +0200
+++ gnome-settings-daemon-3.4.2/configure.ac	2012-05-20 03:37:53.570282529 +0200
@@ -139,8 +139,8 @@
 GUDEV_PKG=""
 AC_ARG_ENABLE(gudev, AS_HELP_STRING([--disable-gudev],[Disable GUdev support]), enable_gudev=$enableval)
 if test x$enable_gudev != xno; then
-	PKG_CHECK_MODULES(GUDEV, gudev-1.0, HAVE_GUDEV="yes", HAVE_GUDEV="no")
-	if test "x$HAVE_GUDEV" = "xyes"; then
+	PKG_CHECK_MODULES(GUDEV, gudev-1.0, have_gudev="yes", have_gudev="no")
+	if test "x$have_gudev" = "xyes"; then
 		AC_DEFINE(HAVE_GUDEV, 1, [define if GUdev is available])
 		GUDEV_PKG="gudev-1.0"
 	else
@@ -151,7 +151,7 @@
 else
 	HAVE_GUDEV=no
 fi
-AM_CONDITIONAL(HAVE_GUDEV, test x$HAVE_GUDEV = xyes)
+AM_CONDITIONAL(HAVE_GUDEV, test x$have_gudev = xyes)
 
 dnl ---------------------------------------------------------------------------
 dnl - common
@@ -314,7 +314,7 @@
 # UPower
 # ---------------------------------------------------------------------------
 PKG_CHECK_MODULES(UPOWER, upower-glib >= $UPOWER_REQUIRED_VERSION libcanberra-gtk3)
-if test x$enable_gudev != xno; then
+if test x$have_gudev != xno; then
 	PKG_CHECK_MODULES(BACKLIGHT_HELPER,
 		glib-2.0 >= $GLIB_REQUIRED_VERSION
 		gudev-1.0
